src/main/java/com/library/java_library_management/dto/BookInfo.java

```diff
@@ -0,0 +1,61 @@
+package com.library.java_library_management.dto;
+
+import com.library.java_library_management.status.BookStatus;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BookInfo {
+    private int book_id;
+    private String author;
+    private String title;
+    private int page_size;
+    private BookStatus status;
```

Comment:
```suggestion
    private final int book_id;
    private final String author;
    private final String title;
    private final int page_size;
    private BookStatus status;
```

status 외에는 setter 를 사용하지 않는 것으로 보입니다. 
그렇다면 status 외에는 setter를 제거하고, final 키워드를 통해 불변 필드로 정의하는 것을 권장드립니다.